### ISSUE TITLE
Matching Colour Themes.

### ISSUE THREAD

**Issue by Penguin-Guru** [vircadia/community-apps#30]
Since these are apps that ship with the client by default, I think they should have matching U.I. For example, Explore has a (nice) dark theme and People has a light theme. Chat is somewhere in between, since it appears to have a black background but a lot of the larger fields are white. Personally, I prefer dark themes. In general, I think pure whites should not be used for interfaces-- very light grey would be my white equivalent. I'm not a very visual person though so don't take my word for it.

**Comment by JulianGro** [comment]
Yeah we kind of need different themes to choose from. Theming and UI in general are pretty controversial topics.
By the way, none of the apps in this repo actually ship with the client by default. 
Explore has it's own repo, and the People app is in the main repo. They are also built with different technologies. Explore is built with HTML, Vue and Javascript, while the People app is built with QML and Javascript.

**Comment by Penguin-Guru** [comment]
I see... Well, it might be good to have one repo for the apps just to make this more intuitive. I was actually planning to make my own changes to the U.I. for some apps but I don't like Javascript so probably not going to happen any time soon. I think it's something best decided by the community anyway since app themes will have big implications on the general game aesthetic. Of course it would be best to have the themes customisable but choosing defaults that match the image you want for the game as a whole is probably important.

As a tangent, I tried clicking the button in Vircadia to make my own app and it just opened instructions (with a white background) on how to add an app to various places. I think it would be good to have at least some info about how to actually make apps on that page, like what languages can be used, or maybe a link to a web page that in turn includes a link to this repo since that window is kind of small.
